sbi/utils/get_models.py

```diff
@@ -2,10 +2,11 @@
 from torch import nn
 from torch.nn import functional as F
 
-from nflows import distributions as distributions_
-from nflows import flows, transforms
-from nflows.nn import nets
-from nflows.nn.nde import MixtureOfGaussiansMADE, MultivariateGaussianMDN
+from pyknos.nflows import distributions as distributions_
+from pyknos.nflows import flows
+from pyknos.nflows import transforms
+from pyknos.nflows.nn import nets
+from pyknos.nflows.nn.nde import MixtureOfGaussiansMADE, MultivariateGaussianMDN
```

Comment:
here `MultivariateGaussianMDN` is imported from `pyknos.nflows.nn.nde` while above it is imported from `pyknos.mde.mde`. what is the difference? Is this correct?